packages/web/src/components/Shelves/Shelf/Shelf.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+// import { useSelector } from 'react-redux';
+import Button from 'react-bootstrap/Button';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLeaf } from '@fortawesome/free-solid-svg-icons';
+// import { getShelf } from '../../../store/shelf/selectors';
+
+import './styles.scss';
+import { Link } from 'react-router-dom';
+import { FlowersList } from '../FlowersList';
+
+type Props = {
+  id: string
+}
+
+export const Shelf = ({
+  id
+}: Props) => {
+  // const {
+  //   isLoading,
+  //   error,
+  //   info
+  // } = useSelector(getShelf);
```

Comment:
Don't forget to remove it.

Reply:
We'll pass it through dev and add the actual logic via redux while merging with another feature